### ISSUE TITLE
SSL issue

### ISSUE THREAD

**Issue by fraxen** [Tropicalista/lucee-cfimap#2]
Hiya - a great little library/tag you have put together - I have still yet to test it thoroughly, but I bumped into a bug...

On my installation (Lucee 4.5/Tomcat/Windows) I had problem doing a secure connect. If I change line 21 in the _connect_ method in _imap/imap.cfc_ to this, then it works:

``` javascript
objProperties.put("mail.store.protocol", arguments.secure ? 'imaps' : 'imap');
```

**Comment by Tropicalista** [comment]
Oh, looks great. I will make a pr asap.

Many thanks

**Comment by Tropicalista** [comment]
Fixed by 94ec89f
